### PR TITLE
Fix Api cennter

### LINKED ISSUE #10634: Swagger changes to align v2 api
According the v2 api changes, we will have 2 version of apis. We need to figure out if we will use 1 swagger yaml file to show both or seperate files. 
Make sure all apis should be showed in the dev denter correctly.

### PR DESCRIPTION
**Pull request by AllForNothing**
Fix #10634 
Env.:https://10.161.79.96:8083
Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/lib/utils/utils.ts [hunk 1]
```diff
@@ -597,3 +597,34 @@ export function formatSize(tagSize: string): string {
...
     }
 }
+
+/**
+ * Simple object check.
+ * @param item
+ * @returns {boolean}
+ */
+export function isObject(item): boolean {
+    return (item && typeof item === 'object' && !Array.isArray(item));
```
Comment (jwangyangls) [inline src/portal/src/lib/utils/utils.ts]: if item is a function?

File: src/portal/src/lib/utils/utils.ts [hunk 2]
```diff
@@ -597,3 +597,34 @@ export function formatSize(tagSize: string): string {
...
+        for (const key in source) {
+            if (isObject(source[key])) {
+                if (!target[key]) { Object.assign(target, { [key]: {} }); }
+                mergeDeep(target[key], source[key]);
+            } else {
+                Object.assign(target, { [key]: source[key] });
+            }
+        }
+    }
+    return mergeDeep(target, ...sources);
```
Comment (jwangyangls) [inline src/portal/src/lib/utils/utils.ts]: Do you need···

File: src/portal/src/lib/utils/utils.ts (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/lib/utils/utils.ts]: will return "function"

File: src/portal/src/lib/utils/utils.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/lib/utils/utils.ts]: needed, because "source"s can be one more objects

**Pull request merged**